app/templates/_base_page.html

```diff
@@ -6,33 +6,50 @@
 {% from "components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "components/tabs/macro.njk" import govukTabs %}
 
+{% set assetPath = '/admin/static' %}
+
 {% block head %}
+  {% include "layouts/_custom_dimensions.html" %}
+  {% include "layouts/_site_verification.html" %}
```

Comment:
Can we split these changes (adding assetPath and adding custom dimensions / site verification) to separate commits?